dockerfiles/platform-build-base/Dockerfile

```diff
@@ -0,0 +1,107 @@
+##############################################################################
```

Comment:
question: @GordonSmith - did you intend to reintroduce this file, having deleted it in https://github.com/hpcc-systems/HPCC-Platform/pull/16876 ?

Reply:
Yes - I suspect there are still scripts referencing it and want to minimize the chances of failures!